src/Controller/Component/CrudComponent.php

```diff
@@ -547,7 +547,7 @@ public function defaults($type, $name, $config = null)
                 $this->config(sprintf('%s.%s', $type, $realName), $config);
             }
 
-            return;
+            return null;
```

Comment:
Your contributions is always welcome, and the majority is accepted as-is. I or others don't always have to agree with all of the changes of a submitted PR. Open Source is not a democracy, far from it. I'm not dictating what is going on in any FoC project, not even my "own" original projects, but I do voice my concerns when i have them. There has been plenty of PRs that i disliked that have gotten merged into FoC projects, and there will continue to be. But I'm against discussions on semantics where there is an IDE or "upstream" conceptual idea on what's corrects or not. Even if it's the cake core. CRUD is a pretty mature project, and tons of people depend on it, and I'm not in favor of breaking apps or preventing people a zero-change upgrade path, just to fix simple things in the "pure" way over the "hack" way when it can be done. e.g. setter/getter vs magic methods. 

a simple change to the Search plugin caused me to spend more than 1 day fixing all our backend apps since a "semantic" change broke the way we used the plugin. I'm sure plenty of people have had that kind of experience.